### ISSUE TITLE
Test data with pooch

### ISSUE THREAD

**Issue by MaceKuailv** [MaceKuailv/seaduck#89]
A version of the following code is used to configure the tests
```
import pooch
pooch_testdata = pooch.create(
    path=pooch.os_cache("seaduck"),
    base_url="doi:10.5281/zenodo.7949168",
    registry=None,
)
pooch_testdata.load_registry_from_doi()  # Automatically populate the registry
```

This result in a "KeyError: "Key"". 

Apparently, folks are already working on this and has already opened a pull request https://github.com/fatiando/pooch/issues/371.

**Comment by MaceKuailv** [comment]
https://github.com/OMS-NetZero/FAIR/pull/138

I know so little about pooch, but this seems like a valid way of getting around the doi issue.

**Comment by MaceKuailv** [comment]
pooch v1.8.0 have this issue resolved. However, the CI process is still downloading some older version at this moment. I will wait a few more days.

**Comment by MaceKuailv** [comment]
closed after github CI updated to pooch 1.8.0
